angular-ngrx-scss/src/app/state/user/user.actions.ts

```diff
@@ -12,3 +13,33 @@ export const fetchUserDataError = createAction(
   '[User API] User Data fetch unsuccessful',
   props<{ error: object }>(),
 );
+
+export const getUserGists = createAction(
+  '[User API] User gists requested',
+  props<{ username: string }>(),
+);
+
+export const getUserTopRepos = createAction(
+  '[User API] User top repos requested',
+  props<{ username: string }>(),
+);
+
+export const getUserTopReposSuccess = createAction(
+  '[User API] User top repos successfully received',
+  props<{ topRepos: UserReposState[] }>(),
+);
+
+export const getUserTopReposError = createAction(
+  '[User API] User top repos unsuccessful',
```

Comment:
Can we change both error action types to  say "fetch unsuccessful", please? The wording without fetch feels a little off.

- [ ] Add "fetch" to both error action types, please